src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -21,29 +22,23 @@ public class SampleDataUtil {
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends"),
-                getCommentSet("Is always cheerful")),
+                new TelegramHandle("alexTelegram"),
+                getTagSet("friends"), getCommentSet("Is always cheerful")),
             new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
-                new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
-                getTagSet("colleagues", "friends"),
-                getCommentSet("Loves candy")),
+                new TelegramHandle("alexTelegram"),
```

Comment:
Everybody's telegram handle is Alex, despite them being different people - is this intentional?